packages/vite/src/client/overlay.ts

```diff
@@ -158,6 +158,12 @@ export class ErrorOverlay extends HTMLElement {
     if (!linkFiles) {
       el.textContent = text
     } else {
+      if (!fileRE.test(text)) {
+        el.textContent = text
+        return
+      } else {
+        fileRE.lastIndex = 0
+      }
```

Comment:
This was added to support none link stack frames like `a non-error was thrown please check your browser's devtools for more information`

this kinda feels hacky. would love any suggestions on how this should be handled